dll/win32/shell32/shresdef.h

```diff
@@ -141,6 +141,7 @@
 #define IDS_CANTSHOWPROPERTIES    159
 #define IDS_CANTDISCONNECT        160
 #define IDS_NONE                  161
+#define IDS_CMD_VERB              162
```

Comment:
Move that down where the other _VERB constants are defined.

Reply:
Moved.